cmd/snap/cmd_get.go

```diff
@@ -77,6 +82,41 @@ func init() {
 		})
 }
 
+type configValue struct {
+	path  string
+	value interface{}
+}
+
+type byConfigPath []configValue
+
+func (s byConfigPath) Len() int           { return len(s) }
+func (s byConfigPath) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
+func (s byConfigPath) Less(i, j int) bool { return s[i].path < s[j].path }
```

Comment:
this will sort things like:

"foo-bar.x" < "foo.bar.x" < "foobar.x"

not sure it's ideal

Reply:
@pedronis Good point.

@stolowski Let's make sure "foo." sorts above any other "foo*"

Reply:
Allright, implemented custom sorting and added a new test - please see TestSortByPath for an extensive example.